src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -1599,9 +1599,10 @@ private void CheckModelAndSyntaxNodeToSpeculate(CSharpSyntaxNode syntax)
                 // Method type parameters are not in scope outside a method
                 // body unless the position is either:
                 // a) in a type-only context inside an expression, or
-                // b) inside of an XML name attribute in an XML doc comment.
+                // b) inside of an XML name attribute in an XML doc comment,
```

Comment:
Just curious, did you compare the pattern you used here with whatever is used already for doc comments?